packages/core/contracts/financial-templates/long-short-pair/LongShortPair.sol

```diff
@@ -93,57 +113,46 @@ contract LongShortPair is Testable, Lockable {
 
     /**
      * @notice Construct the LongShortPair
-     * @param _expirationTimestamp unix timestamp of when the contract will expire.
-     * @param _collateralPerPair how many units of collateral are required to mint one pair of synthetic tokens.
-     * @param _priceIdentifier registered in the DVM for the synthetic.
-     * @param _longToken ERC20 token used as long in the LSP. Mint and burn rights needed by this contract.
-     * @param _shortToken ERC20 token used as short in the LSP. Mint and burn rights needed by this contract.
-     * @param _collateralToken ERC20 token used as collateral in the LSP.
+     * @param params struct to define input parameters to construct the long short pair contract.
      * @param _finder UMA protocol Finder used to discover other protocol contracts.
-     * @param _financialProductLibrary Contract providing settlement payout logic.
-     * @param _customAncillaryData Custom ancillary data to be passed along with the price request. If not needed, this
-     *                             should be left as a 0-length bytes array.
-     * @param _prepaidProposerReward Preloaded reward to incentivize settlement price proposals.
      * @param _timerAddress Contract that stores the current time in a testing environment. Set to 0x0 in production.
      */
     constructor(
-        uint64 _expirationTimestamp,
-        uint256 _collateralPerPair,
-        bytes32 _priceIdentifier,
-        ExpandedIERC20 _longToken,
-        ExpandedIERC20 _shortToken,
-        IERC20 _collateralToken,
+        ConstructorParams memory params,
```

Comment:
nit: why exclude finder and timer?

Reply:
I though it made sence to put the LSP specific contracts in the struct and leave the finder and timer out as the latter two are “supporting” information, where the others are parameterization